README.md

```diff
@@ -1,11 +1,10 @@
 # iesurveykit - Tools for high quality Impact Evaluation Surveys
 
-A repository of survey resourcs
-The iesurveykit is a package of resources for primary data collection. 
+The **`iesurveykit`**  is a package of tools to support high-quality primary data collection.
 
-### Content
-iesurveykit currently supports three major components of the survey workflow survey design; survey completion; and data cleaning and survey harmonization.
-* Data quality checks templates (including real time data quality checks dashboard)
-* Survey completion checks 
+This goes hand in hand with [iefieldkit](https://github.com/worldbank/iefieldkit), a suite of Stata commands related to processing primary data.
 
-This goes hand in hand with [iefieldkit](https://github.com/worldbank/iefieldkit)
\ No newline at end of file
+### Content
+iesurveykit currently supports three major components of the survey workflow: (1) questionnaire design; (2) survey data quality monitoring; and (3) data cleaning and survey harmonization.
```

Comment:
what do we have on (3) for the time being? it would be helpful to link these specifically to the tools/subfolders. e.g. (2) survey data quality monitoring through iesurveychecks

Reply:
on 3 we have the iefieldkit command iecodebook which we can reference here. Included in readme